applications/ColossalQA/data/test_data/test_csv_organization_100.csv

```diff
@@ -0,0 +1,101 @@
+Index,Organization Id,Company Name,Website,Country,Description,Founded,Industry,Number of employees
```

Comment:
Is this file for unit test? If so, please create a folder named `tests`, and put them into this folder

Reply:
it is not for test. will rename it